### PR TITLE
support undefined heading for columns

### PR DESCRIPTION
**Pull request by Alex-Vaky**
This is a fix for issue #122 .

I'm not sure how I feel about the testing additions but I leave that for the review process to guide me to something else if that's desired. 😄

### CODE REVIEW

File: src/app/public/modules/grid/text-trim.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {
+  Pipe,
+  PipeTransform
+} from '@angular/core';
+
+@Pipe({name: 'textTrim'})
+export class TextTrimPipe implements PipeTransform {
```
Comment (Blackbaud-ColbyWhite) [inline src/app/public/modules/grid/text-trim.pipe.ts:7]: wow. i j/ assumed angular had this pipe already in its core 🤯 

very nice 👏 

i'd vote to j/ call it `trim` for simplicity but i don't feel all that passionate one or the other.

**Comment by Alex-Vaky** [comment]
Closing in favor of #125 by @Blackbaud-AlexKingman

**Comment by Blackbaud-AlexKingman** [comment]
@Alex-Vaky apologies! I didn't see this PR until now - that's my mistake. Thanks for your work on this! We should have a release out shortly.

**Comment by Alex-Vaky** [comment]
No worries @Blackbaud-AlexKingman!  Your solution is actually better anyways :P

**Pull request closed without merging**